client/dashboard/dashboard-charts/index.js

```diff
@@ -163,20 +163,16 @@ const DashboardCharts = ( props ) => {
 						hiddenBlocks,
 						onToggleHiddenBlock,
 					} ) }
-					{ window.wcAdminFeatures[
-						'analytics-dashboard/customizable'
```

Comment:
I think I know the answer to this, but just to double check- would we now consider this stable enough to be part of the `analytics` feature flag?